_posts/2020-09-29-sc-election-announce.md

```diff
@@ -0,0 +1,85 @@
+
+The US-RSE Steering Committee sets its own meeting frequency and timing.  Currently, the Steering Committee is meeting every two weeks.  The Steering Committee meeting frequency and schedule for 2021 will be determined once new members are elected.  
+
+The responsibilities of Steering Committee members are detailed in the [US-RSE Membership and Governance](https://github.com/USRSE/documents/blob/master/governance.md) document. They include:
+
+* Setting the overall policy and direction of the US-RSE Association
+* Being responsible for day-to-day activities, operations, and development of the US-RSE Association
+
+Additionally, Steering Committee members are expected to:
+
+* Attend and participate in Steering Committee meetings
+* Follow and participate in Steering Committee discussions on Slack
+* Contribute to US-RSE operations through activities such as
+  * Updating the website
+  * Writing newsletters and communications
+  * Organizing and leading community calls and events
+  * Engaging with community members on Slack
+* Help further the community by 
+  * Participating on international RSE committees
+  * Organizing BOFs, workshops, meetups, and other events at relevant conferences
+  * Leading US-RSE working groups
+  * Writing blog posts
+  * Bringing new ideas to the organization
+
+## Nominations and Candidate Information
+
+Individuals may nominate themself or another member.  In cases when a nomination is not a self-nomination, the Election Chair will contact the nominated individual to confirm their interest in participating. 
+
+During the nomination process, the following information will be required:
+
+**If Nominating Someone Else:**
+
+* Your name and email 
+* Nominee’s name and email 
+* Short statement about why you think the nominated individual will make a good Steering Committee member
+
+Nominations of others will only be shared with the Election Chair and the nominated individual.  The information will not be made public.
+
+**Candidate Form** (Self-nomination or accepted nomination)
+
+* Your name and a publicly-shareable email 
+* A link to a resume, CV, LinkedIn profile, or professional website
+* Why do you want to be on the US-RSE Steering Committee? (<=1000 characters)
+* How do you contribute to the US-RSE Association or RSE community more broadly? (<= 1500 characters)
+
+All candidate information will be shared on the US-RSE website.  Additionally, candidates will have the opportunity to share a short statement (live or recorded) during the US-RSE General meeting, December 4, 2020, 1-3pm ET, on Zoom.  
+
+An #election channel is available on Slack for any discussion with or about candidates.
```

Comment:
This might need to be `#election` so it doesn't render as a heading in markdown.
```suggestion
An `#election` channel is available on Slack for any discussion with or about candidates.
```

Reply:
Actually it looks okay! https://850-120906408-gh.circle-artifacts.com/0/usrse.github.io/2020-09-29-sc-election-announce/index.html We don't need to make the change above, unless you want the text to render differently. Stick out! This overall LGTM.

Reply:
It rendered fine without code quotes